odoo/addons/base/models/ir_sequence.py

```diff
@@ -53,8 +53,14 @@ def _select_nextval(cr, seq_name):
 
 def _update_nogap(self, number_increment):
     number_next = self.number_next
-    self._cr.execute("SELECT number_next FROM %s WHERE id=%%s FOR UPDATE NOWAIT" % self._table, [self.id])
-    self._cr.execute("UPDATE %s SET number_next=number_next+%%s WHERE id=%%s " % self._table, (number_increment, self.id))
+    try:
+        self._cr.execute("SELECT number_next FROM %s WHERE id=%%s FOR UPDATE NOWAIT" % self._table, [self.id])
+        self._cr.execute("UPDATE %s SET number_next=number_next+%%s WHERE id=%%s " % self._table, (number_increment, self.id))
+    except OperationalError as e:
+        if e.pgcode == '55P03':
```

Comment:
While we can't use the more formal errors of 2.8 yet, please check against `psycopg2.errorcodes.LOCK_NOT_AVAILABLE` instead.

Reply:
That's my fault, I had found this: https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/models/ir_cron.py#L245
Indeed, a constant with a clean name is even better

Reply:
> That's my fault 

![200_d](https://user-images.githubusercontent.com/7571158/93332597-58c36b80-f822-11ea-8ce5-1f08e2199087.gif)